src/EventListener/NotifySubscriber.php

```diff
@@ -52,7 +52,7 @@ public function notify(RequestEvent $event): void
     {
         $request = $event->getRequest();
 
-        if (!$event->isMasterRequest()) {
+        if (!$event->isMainRequest()) {
```

Comment:
This doesn't work in SF4. You can use this library if you want: https://github.com/Setono/symfony-main-request-trait